internal/common/testing/dummies.go

```diff
@@ -0,0 +1,12 @@
+package testing
+
+import (
+	"github.com/golang/mock/gomock"
+	"github.com/openshift/assisted-service/internal/stream"
+)
+
+func GetDummyEventStream(ctrl *gomock.Controller) *stream.MockEventNotifier {
+	dummyStream := stream.NewMockEventNotifier(ctrl)
+	dummyStream.EXPECT().Notify(gomock.Any(), gomock.Any()).AnyTimes().Return(nil)
```

Comment:
Is this always what we want or do we want to ensure we're not sending some events in some cases?

Reply:
Ideally we would like to test some cases, but for now I think it's enough (due to high complexity of modifying current tests).
Possibly in a second phase we could implement some real tests about this.